src/App/Pages/Accounts/LoginSsoPage.xaml.cs

```diff
@@ -69,12 +69,12 @@ private void RestoreAppOptionsFromCopy()
             }
         }
 
-        private async void LogIn_Clicked(object sender, EventArgs e)
+        private void LogIn_Clicked(object sender, EventArgs e)
         {
             if (DoOnce())
             {
                 CopyAppOptions();
-                await _vm.LogInAsync();
+                _vm.LogInCommand.ExecuteAsync();
```

Comment:
If you're not going to await it just call `_vm.LogInCommand.Execute(null);` to avoid the warning ([internally](https://github.com/xamarin/XamarinCommunityToolkit/blob/main/src/CommunityToolkit/Xamarin.CommunityToolkit/ObjectModel/Internals/BaseAsyncCommand.shared.cs#L98) it ends up calling `ExecuteAsync`)